core/aws-lsp-fqn/src/node/fqnExtractorWorker.ts

```diff
@@ -0,0 +1,21 @@
+import { parentPort, workerData } from 'worker_threads'
+import { extract } from '../common/fqnExtractor'
+import { FqnExtractorInput } from '../common/types'
+
+async function start() {
+    try {
+        const symbolExtractorInput: FqnExtractorInput = workerData.input
+        console.log(`Extracting fully qualified names for ${symbolExtractorInput.languageId}`)
+
+        const data = await extract(symbolExtractorInput)
+
+        parentPort?.postMessage({ data, success: true })
+    } catch (e) {
+        console.error(`Failed to extract fully qualified names: ${e instanceof Error ? e.message : 'unknown'}`)
```

Comment:
Do you know, will there be any metrics on CW/Toolkit side for us to know how often it works/fails?